vc/credential.go

```diff
@@ -0,0 +1,236 @@
+package vc
```

Comment:
would it make more sense if this file was called `builder.go` given that's what's in it?

Reply:
will rename